python/ray/train/tests/test_backend.py

```diff
@@ -7,11 +7,12 @@
 import time
 
 import ray
+import ray._private.ray_constants as ray_constants
 from ray import train
 from ray.air._internal.util import StartTraceback
 
 # Trigger pytest hook to automatically zip test cluster logs to archive dir on failure
-from ray.tests.conftest import pytest_runtest_makereport  # noqa
+# from ray.tests.conftest import pytest_runtest_makereport  # noqa
```

Comment:
Should this be commented out?

Reply:
Reverted. bazel test didn't work without this comment